run.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
```

Comment:
The fact that we're now building this file in more than 1 repo means is in my opinion a bad thing. You're not thinking about the eventual user. Shouldn't this be in the CLI as part of a streamlined deployment process?

Reply:
We need to be taking learnings from individual blueprints and pulling those learnings into the SDK and CLI in small PRs, so that they become available by default to anyone using the SDK.. including checking tangle network, keystore, no?

Reply:
💯 agree with you, we should upstream this script as a single command in the CLI.